diff -aurN icon-naming-utils-0.6.6.orig/Makefile.am icon-naming-utils-0.6.6/Makefile.am
--- icon-naming-utils-0.6.6.orig/Makefile.am	2006-01-20 18:24:15.000000000 +0100
+++ icon-naming-utils-0.6.6/Makefile.am	2006-01-22 22:35:38.417051000 +0100
@@ -1,5 +1,5 @@
 
-libexec_SCRIPTS = icon-name-mapping
+bin_SCRIPTS = icon-name-mapping
 
 %: %.pl.in
 	sed -e "s#\@PERL\@#$(PERL)#g"		\
@@ -10,19 +10,17 @@
 pkgconfigdir = $(datadir)/pkgconfig
 pkgconfig_DATA = $(pkgconfig_in_files:.pc.in=.pc)
 
-dtddir = $(datadir)/dtds
-dtd_DATA = legacy-icon-mapping.dtd
-
-pkgdata_DATA = legacy-icon-mapping.xml
+pkgdata_DATA =				\
+	legacy-icon-mapping.xml		\
+	legacy-icon-mapping.dtd
 
 EXTRA_DIST =				\
 	icon-name-mapping.pl.in		\
 	$(pkgconfig_in_files)		\
-	$(dtd_DATA)			\
 	$(pkgdata_DATA)
 
 CLEANFILES =				\
-	$(libexec_SCRIPTS)
+	$(bin_SCRIPTS)
 
 DISTCLEANFILES =			\
 	$(pkgconfig_DATA)
diff -aurN icon-naming-utils-0.6.6.orig/icon-naming-utils.pc.in icon-naming-utils-0.6.6/icon-naming-utils.pc.in
--- icon-naming-utils-0.6.6.orig/icon-naming-utils.pc.in	2006-01-20 18:26:56.000000000 +0100
+++ icon-naming-utils-0.6.6/icon-naming-utils.pc.in	2006-01-22 22:48:31.807051000 +0100
@@ -1,6 +1,6 @@
 prefix=@prefix@
 exec_prefix=@exec_prefix@
-program_path=@libexecdir@
+program_path=@bindir@
 
 Name: icon-naming-utils
 Description: Utilities for mapping legacy GNOME and KDE icon names to the new Icon Naming Specification
